src/ComparisonDetails/ComparisonDetails.tsx

```diff
@@ -1,10 +1,16 @@
 import './ComparisonDetails.scss';
+import { Link } from 'react-router-dom';
+import { Comparison } from '../comparisonData'
 
-const ComparisonDetails = () => {
+
+
+const ComparisonDetails = (prop: { selection?: Comparison }) => {
   return(
-    <section>
-      <p>This is ComparisonDetails Component</p>
-    </section>
+      <section>
+        <p>This is ComparisonDetails Component</p>
```

Comment:
The next step here is to add a 'cookie cutter' block of text that we can insert our props and stats into.